### ISSUE TITLE
No binary for esbuild found

### ISSUE THREAD

**Issue by Alduino** [ArtiomTr/aqu#48]
aqu: 0.2.1
node: 16.4.2
pnpm: 6.10.3
os: Pop!_OS (Ubuntu) 21.04

When I try and run `aqu build` or `aqu watch`, it pretty much instantly fails:
```
 Compilation started. 

⠋ Bundling using esbuild[aqu] No binary for esbuild found
 ERROR  Command failed with exit code 1.
```

It looks like this is because it is only testing for `esbuild.exe`. Is aqu compatible with OSs other than Windows?

If I look at where it's looking for (i.e. `/node_modules/.pnpm/aqu@0.2.1_@babel+core@7.14.8/node_modules/esbuild/esbuild.exe`) the `esbuild` binary I have is in `bin/esbuild`, not in the root directory of that package. If I change aqu to look for `esbuild/bin/esbuild` it works (although my code won't compile and esbuild gets a deadlock, but that's probably not an issue with aqu).

**Comment by ArtiomTr** [comment]
Hello @Alduino,

Yes, aqu *should* be compatible with Linux, however I have no way of testing this. In fact, the place where the esbuild binary is located should be decided by esbuild itself. However, there were some troubles with conflicting esbuild versions. Since, as far as I understand, this is not applicable in your case, I will try to add the ability to disable automatic search for binaries.

**Comment by ArtiomTr** [comment]
@Alduino, in the recently released `0.2.2`, the automatic search for esbuild binary is only enabled for the `win32` platform, so your issue should be fixed now.
